src/openrct2/ride/TrackPaint.cpp

```diff
@@ -2167,6 +2168,20 @@ void track_paint(paint_session* session, uint8_t direction, int32_t height, cons
             }
         }
 
+#ifdef __ENABLE_LIGHTFX__
```

Comment:
I'd like to see all of this moved into the light fx file rather than being in the track paint file. It will mean that there are fewer `#define __ENABLE_LIGHTFX___`